<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicClass</class>
 <widget class="QWidget" name="MusicClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="play">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>540</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/assets/playuse.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Music.qrc">
     <normaloff>:/new/prefix1/assets/playuse.png</normaloff>:/new/prefix1/assets/playuse.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>540</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/assets/next.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <disabledoff>:/new/prefix1/assets/next.png</disabledoff>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pre">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>540</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/assets/former.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Music.qrc">
     <normaloff>:/new/prefix1/assets/former.png</normaloff>:/new/prefix1/assets/former.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="dir">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>540</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/assets/extended.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Music.qrc">
     <normaloff>:/new/prefix1/assets/dir.png</normaloff>:/new/prefix1/assets/dir.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="favorite">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>130</width>
     <height>60</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Favorite</string>
   </property>
   <property name="icon">
    <iconset resource="Music.qrc">
     <normaloff>:/new/prefix1/assets/liked.png</normaloff>:/new/prefix1/assets/liked.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="settings">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>130</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
   <property name="icon">
    <iconset resource="Music.qrc">
     <normaloff>:/new/prefix1/assets/setting.png</normaloff>:/new/prefix1/assets/setting.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>550</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>550</y>
     <width>86</width>
     <height>14</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="time_pre">
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="time_aft">
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Music.qrc">
     <normaloff>:/new/prefix1/assets/search.png</normaloff>:/new/prefix1/assets/search.png</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>741</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="rank_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="rank">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>130</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>  Rank</string>
   </property>
   <property name="icon">
    <iconset resource="Music.qrc">
     <normaloff>:/new/prefix1/assets/rank.png</normaloff>:/new/prefix1/assets/rank.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Music.qrc"/>
 </resources>
 <connections/>
</ui>
